Sources/_OpenAPIGeneratorCore/Translator/Responses/translateResponseOutcome.swift

```diff
@@ -87,6 +87,39 @@ extension TypesFileTranslator {
             staticMemberDecl = nil
         }
 
+        let isSingleCaseOutput = operation.containsDefaultResponse && operation.responseOutcomes.count == 1
+
+        var throwingGetterCases: [SwitchCaseDescription] = [
+            SwitchCaseDescription(
+                kind: .case(
+                    .dot(responseKind.identifier),
+                    responseKind.wantsStatusCode ? ["_", "response"] : ["response"]
+                ),
+                body: [.expression(.return(.identifierPattern("response")))]
+            )
+        ]
+        if !isSingleCaseOutput {
+            throwingGetterCases.append(
```

Comment:
```suggestion
        var throwingGetterCases: [SwitchCaseDescription] = [
SwitchCaseDescription(
kind: .case(
.dot(responseKind.identifier),
responseKind.wantsStatusCode ? ["_", "response"] : ["response"]
),
body: [.expression(.return(.identifierPattern("response")))]
            )
        ]
        if !operation.containsDefaultResponse || operation.responseOutcomes.count > 1 {
            throwingGetterCases.append(
```

We can remove the local variable and just inline the check.